source/deployment/sso-saml-adfs.rst

```diff
@@ -0,0 +1,191 @@
+
+	.. image:: ../../source/images/adfs_9_configure_identifiers.PNG
+
+9. In the **Configure Multi-factor Authentication Now** screen, you may enable multi-factor authentication, but this is beyond the scope of this guide.
+
+	.. image:: ../../source/images/adfs_10_configure_mfa.PNG
+
+10. In the **Choose Issuance Authorization Rules** screen, select the option **Permit all users to access this relying party**.
+
+	.. image:: ../../source/images/adfs_11_authorization.PNG
+
+11. In the **Ready to Add Trust** screen, you can review your settings.
+
+	.. image:: ../../source/images/adfs_12_ready_to_add_trust.PNG
+
+12. In the **Finish** screen, select the option **Open the Edit Claim Rules dialog for this relying party trust when the wizard closes**, and click **Close**. You will now exit configuration wizard and a **Claim Rules** editor opens.
+
+	.. image:: ../../source/images/adfs_13_finish_trust.PNG
+
+Create Claim Rules
+------------------
+
+1. In the **Issuance Transform Rules** of the **Claim Rules** editor, click the **Add Rule…** button. This action opens an **Add Transform Claim Rule Wizard**.
+
+	.. image:: ../../source/images/adfs_14_claim_rules_editor.PNG
+
+2. In the **Choose Rule Type** screen, select **Send LDAP Attributes as Claims** from the drop-down menu, then click **Next**.
+
+	.. image:: ../../source/images/adfs_15_choose_rule_type.PNG
+
+3. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and do the following:
+  - From the **LDAP Attribute column**, select ``E-Mail-Addresses``. From the **Outgoing Claim Type**, type ``Email``
+  - From the **LDAP Attribute column**, select ``Given-Name``. From the **Outgoing Claim Type**, type ``FirstName``
+  - From the **LDAP Attribute column**, select `Surname`. From the **Outgoing Claim Type**, type `LastName`
+  - From the **LDAP Attribute column**, select `SAM-Account-Name`. From the **Outgoing Claim Type**, type `Username`
+
+For Mattermost 3.4 and later, the *FirstName* and *LastName* attributes are optional.
+
+Then, click **Finish** to add the rule.
+
+Note that the entries in the **Outgoing Claim Type** column can be chosen to be something else. They can contain dashes but no spaces. Note that they will be used to map the corresponding fields in Mattermost later.
+
+	.. image:: ../../source/images/adfs_16_configure_claim_rule.PNG
+
+4. Create another new rule by clicking the **Add Rule** button.
+
+5. In the **Choose Rule Type** screen, select **Transform an Incoming Claim** from the drop-down menu, then click **Next**.
+
+	.. image:: ../../source/images/adfs_17_transformation_of_incoming_claim.PNG
+
+6. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, then
+  - Select *Name ID* for the **Incoming claim type**
```

Comment:
This file still needs a comprehensive review, but because of time constraints and the fact that there were no changes to content in this PR, which is for adding OneLogin SAML, I only converted to .rst and left reworking for style for another time.